PowerShell/JumpCloud Module/Private/Policies/Dispatch-JCPolicyConfigField.ps1

```diff
@@ -0,0 +1,71 @@
+function Dispatch-JCPolicyConfigField {
```

Comment:
This function is not used, I think we can remove it

Reply:
Agreed